examples/web_testing/BUILD.bazel

```diff
@@ -56,24 +56,7 @@ karma_web_test_suite(
 )
 
 karma_web_test_suite(
-    name = "testing_karma_sauce",
-    browsers = [
-        "@io_bazel_rules_webtesting//browsers/sauce:chrome-win10",
-    ],
-    tags = [
-        "exclusive",
-        "sauce",
-        "native",
-        # TODO(alexeagle): enable on CI once we have set the SAUCE env variables
```

Comment:
do we need to keep this TODO around somewhere? will we ever test with saucelabs?

Reply:
I don't think so. I think saucelabs is going to be a user-space concern as ended up being in the angular repo and not one for the karma_web_test or karma_web_test_suite rule to worry about. The rules_webtesting saucelabs browser target is not very useful.